test/service-object.ts

```diff
@@ -37,8 +43,9 @@ class FakeServiceObject {
 
 describe('GrpcServiceObject', () => {
   // tslint:disable-next-line:variable-name
-  let GrpcServiceObject;
-  let grpcServiceObject;
+  let GrpcServiceObject: typeof ServiceObjectTypes.GrpcServiceObject;
+  // tslint:disable-next-line no-any
+  let grpcServiceObject: any;
```

Comment:
Similar to the last PR, wondering what happens when the annotation is a bit more explicit

Reply:
typescript unhappy, for private function!

Reply:
Gotcha, so as a repeat for records: if the approach for `GrpcService: GrpcService` works out, let's try it with this one too